src/main/java/gameboy/gamepack/racinggame/RacingGamePack.java

```diff
@@ -3,27 +3,28 @@
 import java.util.List;
 
 import gameboy.gamepack.GamePack;
+import gameboy.gamepack.racinggame.data.dto.RaceResultDto;
+import gameboy.gamepack.racinggame.data.entity.Car;
 import gameboy.gamepack.racinggame.data.entity.RacingTrack;
+import gameboy.gamepack.racinggame.data.vo.Name;
 import gameboy.gamepack.racinggame.view.InputView;
 import gameboy.gamepack.racinggame.view.ResultView;
 
 public class RacingGamePack implements GamePack {
 
-    private RacingTrack racingTrack;
-
     @Override
     public void play() {
-        initializeRacingTrack();
+        RacingTrack racingTrack = initializeRacingTrack();
 
-        int raceCount = InputView.inputNumber("시도할 회수는 몇 회 인가요?");
-        for (int i = 0; i < raceCount; i++) {
-            List<Integer> raceResult = racingTrack.startRace();
-            ResultView.display(raceResult);
-        }
+        int raceCount = InputView.inputRaceCount();
+        RaceResultDto raceResult = racingTrack.startRace(raceCount);
+        ResultView resultView = new ResultView();
+        resultView.display(raceResult);
```

Comment:
display가 static 메서드인데 인스턴스화 할 필요는 없겠네요
```suggestion
        ResultView.display(raceResult);
```